### PR TITLE
Scrollable Chat

### LINKED ISSUE #16: Styling for chatbox component

### PR DESCRIPTION
**Pull request by JasperCheung**
This pr adds a scrollable chat. Any additional styling should come in a sep PR.
Issue: Closes #16 

### Frontend:
- `frontend/src/components/ChatMessageList.jsx`: add auto scroll.
- `frontend/src/index.css`: define container size

### CODE REVIEW

File: frontend/src/components/ChatMessageList.jsx [hunk 1]
```diff
@@ -4,17 +4,32 @@ import ChatMessage from './ChatMessage';
...
+
    constructor(props) {
-    super(props);
+     super(props);
+   }
 
+  componentDidUpdate  = () => {
+    this.scrollToMyRef();
   }
+  scrollToMyRef = () => {
```
Comment (joshcwinton) [inline frontend/src/components/ChatMessageList.jsx:16]: fancy af

**Review by joshcwinton — APPROVED** [review]
looks great, maybe we could we add a border to make it more obvious that its scrollable?

File: frontend/src/components/ChatMessageList.jsx (hunk 1 above)
Comment (JasperCheung) [reply frontend/src/components/ChatMessageList.jsx:16]: yes

**Review by joshcwinton — APPROVED** [review]

**Comment by JasperCheung** [comment]
> looks great, maybe we could we add a border to make it more obvious that its scrollable?

added the border, but it does not look good. Lets leave further styling in a different pr and just keep this one as functionality.

**Pull request merged**